### PR TITLE
Add reco material to dqm

### PR DESCRIPTION
**Pull request by rovere**
This will add new plots to constantly monitor the reco-material used.
Validated with the limited matrix for regular and upgrade cases.

### CODE REVIEW

**Comment by ianna** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

File: DQMOffline/Configuration/python/DQMOffline_cff.py [hunk 1]
```diff
@@ -96,17 +96,23 @@
 DQMOfflinePhysics = cms.Sequence( dqmPhysics )
 
 
+from Validation.Geometry.trackingRecoMaterialAnalyzer_cfi import materialDumperAnalyzer
```
Comment (davidlange6) [inline DQMOffline/Configuration/python/DQMOffline_cff.py]: @rovere - would prefer not to have this dependency on validation here - can you move things around to keep DQM independent of Validation (which hopefully it is..). Thanks

Comment (rovere) [inline DQMOffline/Configuration/python/DQMOffline_cff.py]: ciao @davidlange6 ,
you maybe noticed already that there are already other 2 dependencies on validation, besides this new one.
In any case, I'll think about the best approach: either move code in DQM or config in Validation and update this PR.

**Comment by ianna** [comment]
please test

**Comment by rovere** [comment]
I'm not sure the error[1] is related to this PR.


[1]
/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_9_0_X_2017-01-13-2300/external/slc6_amd64_gcc530/lib/libHepMCfio.a: file not recognized: File format not recognized

**Comment by Dr15Jones** [comment]
Please test

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1
(cvmfs problem most likely was the problem behind the first test failures)

**Comment by dmitrijus** [comment]
+1

**Pull request merged**